build.gradle

```diff
@@ -17,16 +17,18 @@
 
 buildscript {
     repositories {
+        google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta3'
```

Comment:
If you update the gradle plugin to beta, Android Studio will force you to keep your plugin's version synced with Android Studio's version. Say you are using Android Studio 3.0.0 beta 6 (latest version), you must also use gradle plugin 3.0.0 beta 6. 
So I think it's better to update the plugin when 3.0.0 stable comes out.

Reply:
All right. no problem.